packages/schema-blocks/tests/blocks/RequiredBlocks.test.ts

```diff
@@ -0,0 +1,109 @@
+import { BlockInstance, createBlock } from "@wordpress/blocks";
+
+import renderer from "react-test-renderer";
```

Comment:
I recently found out why TypeScript gives an error on this import in PHPStorm and how to solve this:
```suggestion
import renderer as * from "react-test-renderer";
```
There is indeed no default export. Importing it like this solves it.

I solved this in my other PR, you may want to do the same here 🙂.

Reply:
I made a mistake, my bad 😬 . But we figured it out.

It should have been:
```js
import * as renderer from "react-test-renderer";
```